extensions/wrapper/wrapper/src/main/java/de/sovity/edc/ext/wrapper/api/ui/pages/contracts/services/utils/OperatorMapper.java

```diff
@@ -0,0 +1,77 @@
+/*
+ *  Copyright (c) 2023 sovity GmbH
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       sovity GmbH - initial API and implementation
+ *
+ */
+
+package de.sovity.edc.ext.wrapper.api.ui.pages.contracts.services.utils;
+import de.sovity.edc.ext.wrapper.api.ui.model.OperatorDto;
+import lombok.RequiredArgsConstructor;
+import org.eclipse.edc.policy.model.Operator;
+
+@RequiredArgsConstructor
+public class OperatorMapper {
```

Comment:
missing test. since mapping should be bijective, you can write one test that tests all values fulfilll A->A'->A with one additional test for any value that ensures that the case does not matter when using fromString

Reply:
OperatorTest was alread there i think, added the string test function